groceries_app/static/css/base.css

```diff
@@ -1,3 +1,25 @@
-nav{
-    margin: 4px 4px 4px 4px;
+nav {
```

Comment:
Curious, why is it nav and not `.nav`?

Reply:
The period is used for classes. Html elements like nav, footer, div, etc. can just be referenced without a period.